openassessment/fileupload/backends/swift.py

```diff
@@ -63,6 +63,10 @@ def get_download_url(self, key):
             )
             raise FileUploadInternalError(ex)
 
+    def remove_file(self, key):
+        """
+        @TODO Need to be implemented to remove previously uploaded files
+        """
```

Comment:
Hmm, yeah that seems worse doesn't it? Let me investigate a bit here, maybe we can implement `remove_file` in swift without too much hassle.